parachains/chain-specs/collectives-polkadot-genesis.json

```diff
@@ -0,0 +1,67 @@
+{
```

Comment:
When spec finalized, it will replace `parachains/chain-specs/collectives-polkadot.json` file.